pkg/utils/rest/error.go

```diff
@@ -0,0 +1,64 @@
+package rest
+
+import (
+	"fmt"
+	"github.com/ablarry/golang-bootcamp-2020/pkg/utils/render"
+	"net/http"
+)
+
+const (
+
+	// Payload error.
+	Payload = "validation.payload"
+)
```

Comment:
By convention, remove the const block if you are only going to declare a single constant.

```suggestion
	// Payload error.
	const Payload = "validation.payload"
```

Also, noticed this constant is only used once, so I think there's no need to use a constant for this.